editor/editor_settings.cpp

```diff
@@ -1144,6 +1149,38 @@ void EditorSettings::setup_language() {
 	load_extractable_translations(lang);
 }
 
+String EditorSettings::get_online_docs_url() {
+	String language = EditorSettings::get_singleton()->get_setting("interface/editor/editor_language");
+	bool override = EditorSettings::get_singleton()->get_setting("interface/editor/override_online_docs_language");
```

Comment:
Why is the boolean setting needed?
Couldn't you instead check if `online_docs_language` is an empty string to disable the override?

Reply:
If we use an empty string to mark the default value, you can't choose an empty string in the dropdown, only by reverting the property: 
![godot windows editor dev x86_64_8j1joxjpbC](https://github.com/user-attachments/assets/ae0e2534-0272-438b-966a-587b99d68688)
I also thought the override bool might be unneeded but I'd prefer an explicit value like `default` or `none` or `auto` or `use_editor_language`.

Reply:
Adding an additional boolean value is redundant. However, having `use_editor_language` means the selection will display that value as is, because string enumeration exports do not support overriding the displayed value. So I'm not sure what the best course of action is.

Reply:
Ah I didn't realize this uses a dropdown to select the editor setting value. Might be best to have it named explicitly, like "auto". Empty looks broken, and the setting itself doesn't say "override" in its name, so you might wonder what it means to have "no language" selected...

Reply:
I like "auto" because the behavior is actually somewhat complex - when set to "auto", it will try to use the main editor languge, then fall back to english. "Use Editor Language", even if it could be displayed in title case rather than snake case, would be technically incorrect. I also did a survey of various other editor and project settings and "auto" is used often enough.

Reply:
Removed the unnecessary boolean, and added an explicitly named "auto" option. Leaving this conversation open to consider other names for the editor setting or the "auto" option.